onnxruntime/core/providers/webnn/builders/op_builder_factory.cc

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Intel Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <vector>
+#include <string>
+
+#include <core/graph/graph.h>
+
+#include "op_builder_factory.h"
+
+namespace onnxruntime {
+namespace webnn {
+
+static OpBuilderRegistrations CreateOpBuilderRegistrations() {
+  OpBuilderRegistrations op_registrations;
+
+  {  // Binary
+    CreateBinaryOpBuilder("Add", op_registrations);
+    CreateBinaryOpBuilder("Sub", op_registrations);
+    CreateBinaryOpBuilder("Mul", op_registrations);
+    CreateBinaryOpBuilder("Div", op_registrations);
+  }
+
+  {  // Activations
+    CreateActivationOpBuilder("Relu", op_registrations);
+    CreateActivationOpBuilder("LeakyRelu", op_registrations);
+    CreateActivationOpBuilder("Sigmoid", op_registrations);
+  }
+
+  {  // Clip
+    CreateClipOpBuilder("Clip", op_registrations);
+  }
+
+  {  // Conv
+    CreateConvOpBuilder("Conv", op_registrations);
+    CreateConvOpBuilder("ConvTranspose", op_registrations);
+  }
+
+  {  // Concat
+    CreateConcatOpBuilder("Concat", op_registrations);
+  }
+
+  {  // Gemm
+    CreateGemmOpBuilder("Gemm", op_registrations);
```

Comment:
`CreateGemmOpBuilder("MatMul", op_registrations);` too? I see MatMul checked here: https://github.com/microsoft/onnxruntime/pull/15698/files#diff-80ffe78c84984a47483ee44069d532504c82ccc174aaa5a36dfa8900c3530662R40

Reply:
Oops, I forgot to remove MatMul check, as which hasn't been implemented in Chromium.

Reply:
Doh. Too bad because I have MatMul implemented in my fork :b. [https://github.com/fdwr/chromium-src-webnn-dml/pull/1/files#diff-48bafbfc616dd59a5134cc609ccd6936f00b36e8ad814e950f8f235a9980bf21R2279](https://github.com/fdwr/chromium-src-webnn-dml/pull/1/files#diff-48bafbfc616dd59a5134cc609ccd6936f00b36e8ad814e950f8f235a9980bf21R2279)